hack/quickstart-setup.sh

```diff
@@ -462,7 +462,7 @@ success "Kuadrant sample configuration deployed."
 
 # Deploy gateway
 info "Deploying example gateway ..."
-kustomize build config/dependencies/istio/gateway | kubectl apply -f -
+kustomize build ${KUADRANT_REPO_RAW}/config/dependencies/istio/gateway | kubectl apply -f -
```

Comment:
I haven't tried this but I am always wondering why write this in this fashion, why use a pipe?
``` sh
kubectl apply -k ${KUADRANT_REPO_RAW}/config/dependencie/istio/gateway
```

Reply:
I believe it's because it's the output of kustomize build that's needed rather than the raw directory.

Reply:
This I understand, the raw directory will fail if the `-f` flag is used. In my example it is the ` -k / --kustomize ` flag that is used. This requires a kustomization directory and under the hood will run kustomize build. 

If you try using a file with the `-k` it will fail, like how `-f` fails with a directory.

Reply:
ah okay. I think that's what @david-martin is referring to also here -> https://github.com/Kuadrant/docs.kuadrant.io/pull/161#discussion_r1894023740 so I will make those updates as part of this pr. 👍

Reply:
ack

Reply:
These changes are ready for review now.